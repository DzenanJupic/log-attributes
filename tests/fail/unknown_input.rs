#[log_attributes::log(info, "{fn} was executed with {a}")]
fn unknown_input() {}

fn main() {}
