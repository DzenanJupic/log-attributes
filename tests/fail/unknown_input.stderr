error[E0425]: cannot find value `a` in this scope
 --> tests/fail/unknown_input.rs:1:1
  |
1 | #[log_attributes::log(info, "{fn} was executed with {a}")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope
  |
  = note: this error originates in the attribute macro `log_attributes::log` (in Nightly builds, run with -Z macro-backtrace for more info)
